plugins/modules/zos_copy.py

```diff
@@ -1331,9 +1369,16 @@ def _mvs_copy_to_uss(
                     os.mkdir(dest)
                 except FileExistsError:
                     pass
+        opts = dict()
+        if self.executable:
+            opts["options"] = "-IX"
+
         try:
             if src_member or src_ds_type in data_set.DataSet.MVS_SEQ:
-                response = datasets._copy(src, dest)
+                if self.executable:
+                    response = datasets._copy(src, dest, None, **opts)
```

Comment:
I would advocate for keeping this simple, the line
```response = datasets._copy(src, dest, None, **opts)```
Doesn't need to distinguish between when self.is_executable is True or False, if it is True opts will contain -X option but if is False ops will be empty. So rather just to keep that if in line 1350 than having two.

Reply:
Ready